jenkins.sh

```diff
@@ -10,5 +10,7 @@ pip2.7 install sphinx
 make develop extras=[aws,mesos,azure,encryption,cwl]
 export LIBPROCESS_IP=127.0.0.1
 export PYTEST_ADDOPTS="--junitxml=test-report.xml"
+rm -rf /mnt/ephemeral/tmp
```

Comment:
Ahh yes, the -f does that.

However, to prove that it doesn't fail if /mnt/ephemeral/tmp is absent, you would need to run it on a fresh slave.